lib/public/views/Logs/Create/LogCreationModel.js

```diff
@@ -123,6 +173,7 @@ export class LogCreationModel extends Observable {
         try {
             const { data: createdLog } = await jsonFetch('/api/logs', options);
             this._createdLog = RemoteData.success(createdLog);
+            this._submitted = true;
```

Comment:
Please use `createdLog` instead of adding a new flag

Reply:
refactored